### PR TITLE
Return empty string when Accept-Encoding is undefined

### PR DESCRIPTION
**Pull request by cakecatz**

### CODE REVIEW

**Comment by cakecatz** [comment]
Maybe #78 is related

File: server.coffee [hunk 1]
```diff
@@ -219,7 +219,7 @@ server = Http.createServer (req, resp) ->
       'Via'                     : user_agent
       'User-Agent'              : user_agent
       'Accept'                  : req.headers.accept ? 'image/*'
-      'Accept-Encoding'         : req.headers['accept-encoding']
```
Comment (atmos) [inline server.coffee]: Should we set a sane default here instead of the empty string?

**Comment by atmos** [comment]
Looking at the spec it looks like an empty string is fine.

**Comment by atmos** [comment]
Thanks

**Pull request merged**